### DISCUSSION TITLE
No Docker images?

### DISCUSSION THREAD

**Discussion by Kurty00** [zws-im/zws#690] [Q&A] [answered]
Is there a reason docker images are no longer supplied?

**Comment by jonahsnider** [comment, marked as answer]
Supporting self-hosting in ZWS ended up limiting the speed I could add features & iterate, since it would cause breaking changes for self-hosters. It's still possible to self-host, just more annoying to do so now. You should be able to build a Docker image by cloning the repo and running `yarn install` and then `yarn build:nixpack`, which will output a Docker image to the `zws-im/zws` tag. Note that this will run the full frontend & backend. There is no way to separate them without making it more annoying for me to run the actual ZWS production instance.
